pkg/volume/plugins.go

```diff
@@ -455,6 +455,7 @@ type Spec struct {
 	Volume           *v1.Volume
 	PersistentVolume *v1.PersistentVolume
 	ReadOnly         bool
+	InlineVolume     bool
```

Comment:
I think that the name is misleading - in-tree volume plugins may get an impression that this distinguishes between `Volume` and `PersistentVolume`, which is not the case here.

Reply:
Changed to `InlineVolumeSpecForCSIMigration`